<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServeurWidget</class>
 <widget class="QWidget" name="ServeurWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServeurWidget</string>
  </property>
  <widget class="QPushButton" name="pushButtonLancementServeur">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Lancement serveur</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelClients">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Clients</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuitter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="labelPort">
      <property name="text">
       <string>Num√©ro de Port</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBoxPort">
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>8888</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listWidgetClients">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>ServeurWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
